capstone-backend/src/test/java/com/google/sps/CommunityTest.java

```diff
@@ -33,7 +36,6 @@ public void setUp() {
   public void testConstructor() {
     Assert.assertEquals(community.getName(), NAME);
     Assert.assertEquals(community.getDescription(), DESCRIPTION);
-    Assert.assertEquals(community.getAnnouncement(), "");
```

Comment:
I think this was the line that was causing the test to fail that Mauro is talking about? This test is failing on my machine but the getAnnouncement() line it is talking about isn't there, so I think it's fine.